docs/static/faq.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[faq]] 
+== Frequently Asked Questions (FAQ)
+
+We will be adding more questions and answers, so please check back soon.
```

Comment:
should we link to this file on github and ask folks to open prs about questions they think we should have here?

Reply:
I like that idea.  After I publish this topic, I'll rework the PR links directly to the source file. For now, I linked it to the repo as a placeholder.